From: "David S. Miller" <davem@davemloft.net>
Date: Mon, 7 Oct 2019 09:37:27 -0400
Subject: ipv6: Make ipv6_mc_may_pull() return bool.
Patch-mainline: v5.5-rc1
Git-commit: 328908621081c3c7455c39549c5334e74b7c525a
References: bsc#1154353

Consistent with how pskb_may_pull() also now does so.

Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 include/net/addrconf.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/include/net/addrconf.h
+++ b/include/net/addrconf.h
@@ -202,11 +202,11 @@ u32 ipv6_addr_label(struct net *net, con
 /*
  *	multicast prototypes (mcast.c)
  */
-static inline int ipv6_mc_may_pull(struct sk_buff *skb,
-				   unsigned int len)
+static inline bool ipv6_mc_may_pull(struct sk_buff *skb,
+				    unsigned int len)
 {
 	if (skb_transport_offset(skb) + ipv6_transport_len(skb) < len)
-		return 0;
+		return false;
 
 	return pskb_may_pull(skb, len);
 }
